tests/models/test_momentum_balance.py

```diff
@@ -675,12 +679,12 @@ def test_time_dependent_bc():
 
     # Fixed values from a previous run. Both normal value (u_y=0) and ratio of
     # tangential displacements (1/50, see BC class) are correct.
-    u_e = np.array([0.50754939, 0.0, 25.37756547])
```

Comment:
I trust that the change of values is reasonable here, but it seems that documentation on the surrounding lines (both above and below) should be updated. Please consider this.

Reply:
The documentation is updated now.